builder/image/slugbuilder/builder/install-buildpacks

```diff
@@ -8,10 +8,17 @@ download_buildpack() {
     buildpack_name=$(basename $buildpack_url)
     buildpack_commit="$2"
 
-    git clone $buildpack_url $BUILDPACK_INSTALL_PATH/$buildpack_name
-    pushd $BUILDPACK_INSTALL_PATH/$buildpack_name 2>&1 >/dev/null
-        git checkout --quiet $buildpack_commit
-    popd 2>&1 >/dev/null
+    set +e
+    git clone --branch $buildpack_commit --depth 1 $buildpack_url $BUILDPACK_INSTALL_PATH/$buildpack_name
```

Comment:
`--branch` doesn't like commits, hence the `git checkout`:

```
><> git clone --branch 6e79094 --depth 1 https://github.com/ddollar/heroku-buildpack-multi.git
Cloning into 'heroku-buildpack-multi'...
warning: Could not find remote branch 6e79094 to clone.
fatal: Remote branch 6e79094 not found in upstream origin
```

I guess this will only work against tagged versions of a buildpack? Some of them do not use tags at all.